tests/pmap_test.py

```diff
@@ -1324,12 +1324,6 @@ def testJitPmapComposition(self):
     ans = pmap(jit(f), 'i')(x)
     self.assertAllClose(ans, expected, check_dtypes=False)
 
-  def testMakeJaxprOfOpenSpmd(self):
-    f = lambda x: x - lax.psum(x, 'i')
-    shape = (xla_bridge.device_count(), 4)
-    x = np.arange(prod(shape), dtype=np.float32).reshape(shape)
-    make_jaxpr(f)(x)  # doesn't crash
```

Comment:
I removed this test, because it was already quite fictional. For example, replacing `lax.psum(x, 'i')` with `lax.psum(1, 'i')` makes it fail with an unbound axis error. If we really want to give people the ability to trace jaxprs with names, then we should ask them to provide the environments to `make_jaxpr`.

Reply:
Maybe do a global presubmit to see if someone is relying on this behavior, and fix downstream?